src/plugin-host/plugin-events/event-arguments/leaderboard-reset-plugin-event-arguments.ts

```diff
@@ -0,0 +1,25 @@
+import { PluginEventArguments } from "../plugin-event-arguments";
+import { Chat } from "../../../chat/chat";
+
+/**
+ * Leaderboard Reset Plugin Event Arguments.
+ * Contains chat that has been reset.
+ */
+export class LeaderboardResetPluginEventArguments extends PluginEventArguments
```

Comment:
I can see reasons for this also including the last generated leaderboard.

Reply:
Maybe if that becomes an actual requirement this can be implemented.